crux-llvm/svcomp/README.md

```diff
@@ -108,4 +108,148 @@ $ PYTHONPATH=../benchexec python3 -m benchexec.benchexec -r SV-COMP21_unreach-ca
+3. Use `benchexec` to run each of the witness validators. I have checked in a
+   `scripts/validate-witnesses.sh` script that I use to automate this process.
+4. Each validators' `results/` directories will now have information about
+   witness validity that needs to be pieced together. The `benchexec` repo has
+   a [`mergeBenchmarkSets.py`](https://github.com/sosy-lab/benchexec/blob/344eb314bad07142a8ff0cebc2eae9502470ef60/contrib/mergeBenchmarkSets.py)
+   script that does this for you. This script takes the original run's
+   `.xml.bz2` file (containing the results of the run) as its first argument,
+   and each subsequent argument is an `.xml.bz2` file containing results from
+   a witness validator. Here is one example of how to invoke it:
+
+   ```bash
+   PYTHONPATH=benchexec-master python3 -m contrib.mergeBenchmarkSets /home/bench-defs/crux-llvm-svcomp-2021-09-15-Linux-x86_64/results/crux.2021-09-16_02-25-33.results.SV-COMP21_no-overflow.xml.bz2 /home/bench-defs/val_cpa-seq/results/cpa-seq-validate-correctness-witnesses.2021-09-21_19-28-15.results.SV-COMP21_no-overflow.xml.bz2 /home/bench-defs/val_cpa-seq/results/cpa-seq-validate-violation-witnesses.2021-09-21_20-48-27.results.SV-COMP21_no-overflow.xml.bz2 /home/bench-defs/val_cpa-witness2test/results/cpa-witness2test-validate-violation-witnesses.2021-09-21_21-36-02.results.SV-COMP21_no-overflow.xml.bz2 /home/bench-defs/val_fshell-witness2test/results/fshell-witness2test-validate-violation-witnesses.2021-09-21_21-50-30.results.SV-COMP21_no-overflow.xml.bz2 /home/bench-defs/val_metaval/results/metaval-validate-correctness-witnesses.2021-09-21_21-59-19.results.SV-COMP21_no-overflow.xml.bz2 /home/bench-defs/val_metaval/results/metaval-validate-violation-witnesses.2021-09-22_01-25-21.results.SV-COMP21_no-overflow.xml.bz2 /home/bench-defs/val_uautomizer/results/uautomizer-validate-correctness-witnesses.2021-09-22_02-20-16.results.SV-COMP21_no-overflow.xml.bz2 /home/bench-defs/val_uautomizer/results/uautomizer-validate-violation-witnesses.2021-09-22_09-45-00.results.SV-COMP21_no-overflow.xml.bz2
+   ```
+
+   This produce a `crux.<date>.results.<benchmark-definition>.xml.bz2.merged.xml.bz2`
+   file in the same `results/` directory as the original
+   `crux.<date>.results.<benchmark-definition>.xml.bz2`. Unfortunately, it does
+   not produce a corresponding `.txt` file containing the total score, so this
+   must be computed manually. I have checked in a `scripts/CountScore.hs`
+   script that I use to automate this process. It will output something like
+   this:
+
+   ```bash
+   $ ./CountScore.hs /home/rscott/bench-defs/crux-llvm-svcomp-2021-09-15-Linux-x86_64/results/crux.2021-09-16_02-25-33.results.SV-COMP21_no-overflow.xml.bz2.merged.xml.bz2
+   Statistics:                     631 Files
+     correct:                      249
+       correct true:               61
+       correct false:              188
+     correct-unconfirmed:          249
```

Comment:
Looks like this is just the same count as the `correct` category.  I'd expect to se `28` here.

Reply:
Oops, that's an embarrassing mistake. Fixed.